<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactDetectionTestPlugin</class>
 <widget class="QDockWidget" name="ContactDetectionTestPlugin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>698</height>
   </rect>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Contact Detection Test Plugin</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>317</width>
         <height>653</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QGroupBox" name="statusBox">
          <property name="title">
           <string>Status</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="status">
             <property name="text">
              <string>Please select which test to perform.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QToolBox" name="toolBox">
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="selectionBox">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>299</width>
             <height>483</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Test Selection</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="2" column="0">
             <widget class="QListWidget" name="testList"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="testListLabel">
              <property name="text">
               <string>Test:</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="parameterBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>299</width>
             <height>483</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Input Parameters</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout"/>
          </widget>
          <widget class="QWidget" name="executionBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>299</width>
             <height>483</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Test Execution</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QGroupBox" name="posesBox">
              <property name="title">
               <string>Predefined Poses</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QListWidget" name="poses">
                 <property name="editTriggers">
                  <set>QAbstractItemView::NoEditTriggers</set>
                 </property>
                 <property name="selectionMode">
                  <enum>QAbstractItemView::SingleSelection</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="resultsBox">
              <property name="title">
               <string>Operations</string>
              </property>
              <layout class="QFormLayout" name="formLayout">
               <item row="2" column="0">
                <widget class="QPushButton" name="_detectBtn">
                 <property name="text">
                  <string>New Detection</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="0" colspan="2">
                <widget class="QPushButton" name="_verbose">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Show Verbose Output</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QPushButton" name="_trackBtn">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Track</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QPushButton" name="_trackDetectBtn">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Track + Detect</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QCheckBox" name="_autoDetect">
                 <property name="toolTip">
                  <string>Check to automatically repeat operation when state is changed in RobWorkStudio.</string>
                 </property>
                 <property name="text">
                  <string>Auto-detect</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="contactsGroup">
              <property name="title">
               <string>Contacts</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="0" column="0">
                <widget class="ContactTableWidget" name="_contacts"/>
               </item>
               <item row="1" column="0">
                <widget class="QPushButton" name="_clear">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Clear</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ContactTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">rwsimlibs/gui/ContactTableWidget.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
